integration_tests/src/main/python/spark_session.py

```diff
@@ -71,10 +71,14 @@ def _set_all_confs(conf):
         _spark.conf.set("spark.rapids.sql.test.injectRetryOOM", "true")
     else:
         _spark.conf.set("spark.rapids.sql.test.injectRetryOOM", "false")
+    _spark.conf.set("spark.rapids.sql.test.isTestRun", "true")
```

Comment:
why do we need a new config? We already have a testing config.

Reply:
@revans2 and I discussed this one offline. This was an optimization to not instantiate the `RapidsConf` for each batch. We discussed changing this so we can memoize the config per task instead, so I should be able to remove this config since the overhead should be much less.

Note that the other test config we have is: `spark.rapids.sql.test.enabled` which is more around verifying that all operators in a SQL query are on the GPU, so we assert that we have replaced everything and else the tests should fail. That said, less configs is better so I am all for simplifying this.